TypeCobol.Test/Misc/ExpressionsTokens-expected.txt

```diff
@@ -0,0 +1,48 @@
+Line 24: X =  Z
+Line 28: X equal Z
+Line 32: X not =  Z
+Line 36: X not equal Z
+Line 40: X <  Y
+Line 44: X less Y
+Line 48: X >  Y
+Line 52: X greater Y
+Line 56: X <=  Y
+Line 60: X less Y
+Line 64: X not greater Y
+Line 68: X >=  Y
+Line 72: X greater Y
```

Comment:
This is not a bug, the 2 expressions are different and the builder correctly creates the operator. However this test uses tokens from the expressions to render the result. When the operator is defined using multiple tokens, only the first one is captured in the operator instance so both `greater than` and `greater than or equal to` are rendred as 'greater'.

Maybe this could be fixed using a TokenGroup ?

Reply:
Ha ok.
I've created #2334. Yes TokenGroup seems a good solution.

Reply:
Directly included in this PR, I suggest to squash and use a commit message mentioning both work items.